content/hints.md

```diff
@@ -0,0 +1,127 @@
+
+  If your submission is incorrect,
+  the error message you get will be the error that occurred
+  on the first input file on which you failed.
+  E.g., if your submission is prone to crash but also incorrect,
+  your submission may be judged as either "Wrong Answer" or "Run Time Error",
+  depending on which is discovered first.
+
+- Your submission will have exactly one processor core
+  fully at its disposal while running.
+  You should not attempt to use multithreading in your submissions.
+  Attempts to start extra processes or threads may
+  lead to a judgement of "Run Time Error".
+
+- Each problem will have a stated time limit.
+  This time limit is per input file and refers to CPU time,
+  which includes time spent on slow parsing
+  (e.g., using `Scanner` in Java).
+
+- All judging machines are physically identical
+  to the laptops provided to you during the contest
+  (see [Systems](/systems)).
+
+- The memory limit is the same for all problems and is 2048 MB.
+
+- For problems with large I/O,
+  it is in your best interest to use buffered input and output in Java.
+  We have tried to make sure that the I/O-intensive problems
+  are solvable using even slow I/O.
+  But for some problems it can be more difficult to get accepted
+  using e.g. the `Scanner` class,
+  since it consumes a significant amount of valuable CPU time
+  that could otherwise be used by your algorithm.
+  In most cases, a `BufferedReader` will be more efficient.
+  Also consider explicit output buffering
+  (e.g., by using the `BufferedWriter` class)
+  to avoid flushing every line of output separately.
+
+- We guarantee that every problem is solvable in C++, Java, and Python 3.
+  There is no such guarantee for Kotlin.
+
+
+## Problem Statements
+
+We generally try to make the input and output formats
+as easy as possible to follow for you as contestants —
+input and output are meant to be the easiest parts of the problems!
+
+- For input, multiple numbers or words on one input line
+  will be separated by single spaces.
+  There are no additional spaces, tabs or newlines.
+
+- For output, unless explicitly stated in the problem statement,
+  we do not require you to output
+  exactly the right number of spaces and newlines.
+  Errors in whitespace within reason will be accepted,
+  but if all whitespace between two tokens are omitted,
+  that is (of course) an error.
+
+- For problems with real-valued output,
+  we generally only require that your output is correct
+  up to a certain absolute or relative error.
+  In the case that both are specified, the largest of the two applies.
+  For example, if the problem statement requests a
+  "absolute or relative error of at most $10^{-6}$", this means that:
+
+  - If the correct answer is $0.005$, any answer between $0.004999$ and $0.005001$ will be accepted.  
+    The absolute error of $\pm 10^{-6}$ is larger than the relative error of $\pm 5 \cdot 10^{-9}$ ($= 0.005 \cdot 10^{-6}$).
```

Comment:
Latex doesn't work in hugo.

Reply:
Woops, I forgot to commit the MathJax `<script>` that I put in the footer :joy: Should be fixed now :slightly_smiling_face: On my machine, it looks like this:
![image](https://user-images.githubusercontent.com/9739541/199088659-d475b7bd-175c-4d27-b999-e9b0679d6123.png)

Reply:
On second thought: I've changed Mathjax to a shortcode instead of always being included from `layouts/partials/site-footer.html`. I noticed that, on the Systems page, it would mess with the dollar signs in the compilation commands :joy: